olp-cpp-sdk-dataservice-read/src/repositories/PrefetchTilesRepository.cpp

```diff
@@ -44,7 +44,7 @@ namespace repository {
 
 namespace {
 constexpr auto kLogTag = "PrefetchTilesRepository";
-constexpr std::uint32_t kMaxQuadTreeIndexDepth = 4u;
+constexpr std::int32_t kMaxQuadTreeIndexDepth = 4u;
```

Comment:
?
I mean change the value then as well. And this file has quite a lot of conversion issues anyway.